packages/components/src/angle-picker/index.js

```diff
@@ -0,0 +1,112 @@
+/**
```

Comment:
Merge issue? It looks like it was renamed: https://github.com/WordPress/gutenberg/tree/master/packages/components/src/angle-picker-control.

Reply:
It looks like a merge issue, removed the folder 👍